recipes/acme_dns_server.rb

```diff
@@ -0,0 +1,80 @@
+#
+# Cookbook:: osl-acme
+# Recipe:: acme_dns_server
+#
+# Copyright:: 2017-2021, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package 'glibc.i686'
+
+user 'acme-dns' do
+  system true
+end
+
+directory '/etc/acme-dns' do
+  recursive true
+  action :create
+end
+
+acme_dns_version = node['osl-acme']['acme-dns']['version']
+remote_file '/etc/acme-dns/acme-dns.tar.gz' do
```

Comment:
This file should be downloaded to this location, *not* in `/etc`.
```suggestion
remote_file "#{Chef::Config[:file_cache_path]}/acme-dns.tar.gz" do
```